src/hackerspace_online/settings.py

```
@@ -555,9 +562,14 @@
 LOGIN_URL = 'account_login'
 ACCOUNT_ADAPTER = "hackerspace_online.adapter.CustomAccountAdapter"
 # Specifies the adapter class to use, allowing you to alter certain default behaviour.
-ACCOUNT_AUTHENTICATION_METHOD = "username"  # (=”username” | “email” | “username_email”)
+ACCOUNT_LOGIN_METHODS = {"username"}  # ({"username"} | {"email"} | {"username", "email"})
 # Specifies the login method to use – whether the user logs in by entering their username,
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)